grails-plugin-gsp/src/main/groovy/org/grails/plugins/web/GroovyPagesGrailsPlugin.groovy

```diff
@@ -202,7 +202,7 @@ class GroovyPagesGrailsPlugin extends Plugin {
 
         // Setup the main templateEngine used to render GSPs
         groovyPagesTemplateEngine(GroovyPagesTemplateEngine) { bean ->
-            classLoader = ref("classLoader")
+            bean.lazyInit = true
```

Comment:
Is this change related to the issue?

Reply:
no. gspTagLibraryLookup is very time consuming, it is lazy, so I add too.

Reply:
Based on my test, the lazy-init doesn't have a noticeable effect, I can remove it currently.

Reply:
I found a method to optimize `gspTagLibraryLookup`, this will greatly reduce initialization time.